stable/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 1.1.0
+version: 1.1.1
```

Comment:
This is a new feature, so that would be `x.y+1.z` ?

Reply:
Good point. I will fix that.